CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR)
+
+project(TorchRec
+        VERSION 0.2.0
```

Comment:
no need for version, scikit build should take care of it

Reply:
removed.